tests/test_config.py

```diff
@@ -1868,6 +1868,46 @@ def test_quick_search_model_specific(self):
         with self.assertRaises(TritonModelAnalyzerException):
             self._evaluate_config(args, yaml_content, subcommand='profile')
 
+    def test_model_weightings(self):
+        """
+        Test that weightings can be specified only globally in the YAML
+        """
+        args = [
+            'model-analzyer', 'profile', '--model-repository', 'cli-repository',
+            '--run-config-search-mode', 'quick', '-f', 'path-to-config-file'
+        ]
+
+        # Global weightings - legal
+        yaml_content = """
+        weightings: [3, 1]
```

Comment:
This does not seem very use friendly to me. The name is very generic and doesn't specify what kind of weights they are (model weights). Also, the user has to know that the order of the values corresponds to the order of the keys in the below profile_models.
Is there any reason we can't just specify a model weight (a single integer value) inside of the model like we do for objectives and constraints, with a default of 1?

Reply:
I've changed `weightings` to `weighting` and switched to now flag this if it is set globally. We now take an integer (instead of a list) and the default value is 1 if not specified for a model.